src/pybind/mgr/dashboard/services/cluster.py

```diff
@@ -44,6 +49,13 @@ def from_db(cls):
     @classmethod
     def get_capacity(cls) -> ClusterCapacity:
         df = mgr.get('df')
+        total_objects = 0
+        total_pool_bytes_used = 0
+        for pool in df['pools']:
+            total_objects = total_objects + pool['stats']['objects']
+            total_pool_bytes_used = total_pool_bytes_used + pool['stats']['bytes_used']
```

Comment:
Let's rather use list comprehensions/generators and reducers (`sum`):
```suggestion
        total_objects = sum(pool['stats']['objects'] for pool in df['pools'])
        total_pool_bytes_used = sum(pool['stats']['bytes_used'] for pool in df['pools'])
```

Reply:
We have modified the code a bit, we have an if condition running in the for loop as well..not sure if we can do it this way